cpp/src/groupby/sort/group_scan_util.cuh

```diff
@@ -26,6 +26,7 @@
 #include <cudf/detail/gather.hpp>
 #include <cudf/detail/iterator.cuh>
 #include <cudf/detail/null_mask.hpp>
+#include <cudf/detail/structs/utilities.hpp>
```

Comment:
Why?

Reply:
https://github.com/rapidsai/cudf/blob/dc3c1a884363c97e8a3232dd0e3b928937bf0d0c/cpp/src/groupby/sort/group_scan_util.cuh#L230

This was being transitively included before

Reply:
Oh but now `struct_minmax_util` already included it so we can remove this now.